main/EditText/index.tsx

```diff
@@ -37,6 +37,7 @@ const StyledRowInput = styled.TextInput`
   flex: 1;
   height: 100%;
   color: #2c374e;
+  ${Platform.OS === 'web' && { 'outline-style': 'none' }}
```

Comment:
@DevYuns I think you can tell the effect of this in more detail. Maybe some screenshots.

Reply:
TextInput element already has border-width as you can see in line 397. But when clicking the TextInput element(I mean when element is focused), It show two line overlapped. one is focus outline, the other is border-width in web(The focus outline doesn't appear in mobile). So I add this code. It make focus-outline invisible in web. :)

Reply:
<img width="447" alt="Screen Shot 2020-06-17 at 9 53 32" src="https://user-images.githubusercontent.com/58724686/84842607-87de0b80-b080-11ea-9ddd-6e6195a0a4a3.png">